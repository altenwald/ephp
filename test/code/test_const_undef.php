<?php

print MAX_INT . "\n";

