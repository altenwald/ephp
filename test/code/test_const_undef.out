
Notice: Use of undefined constant MAX_INT - assumed 'MAX_INT' in {{CWD}}/test/code/test_const_undef.php on line 3
MAX_INT
